x/valset-pref/validator_set.go

```diff
@@ -102,6 +131,9 @@ func (k Keeper) UndelegateFromValidatorSet(ctx sdk.Context, delegatorAddr string
 	}
 
 	// Handles rounding for ex: 9999.99999 = 10000.0
+	// Matt: Can we have test cases that handles this rounding?
```

Comment:
Yup for this, i think rounding is not really necessary. I removed and added test case